device/samples/readme.md

```diff
@@ -154,7 +154,7 @@ node sample_sample_device.js
 
 | Sample                  | Description                                                           | JavaScript | TypeScript |
 | ----------------------- | --------------------------------------------------------------------- | :--------: | :--------: |
-| upload_to_blob          | Uploads a blob to Azure through IoT Hub                               |     ✔      |     ✔      |
+| upload_to_blob          | Uploads a blob to Azure through IoT Hub (deprecated)                             |     ✔      |     ✔      |
```

Comment:
suggestion: remove this sample entirely. Let's steer our users away from this by just not offering this sample. They can still use the API obviously, but offering them a sample for something we don't want them to use is like... well not sure what it's like but it is a bad idea.

Reply:
@YoDaMa I am going to keep the sample in there but deprecated for a few weeks instead of just removing it. But I agree, we can then remove it entirely.